### DISCUSSION TITLE
Forgot to renew my domain.

### DISCUSSION THREAD

**Discussion by manangouhari** [vercel/community#4412] [Help]
### Summary

I've yet again forgotten to renew my domain which has expired and there's no way to renew it myself.

Hence, reaching out to Vercel support. Kindly help me renew my domain if possible. 

### Example

manangouhari.com

### Steps to Reproduce

Visit the [https://manangouhari.com](domain). It has expired.

**Comment by lucas-0liveira** [comment]
I'm sorry to hear that. It's a common thing to forget, so no worries there. It's great that you're reaching out to Vercel support for assistance. They should be able to help you with the renewal process. Just let them know about the situation and they'll guide you through the steps to get your domain renewed. Good luck!

**Reply by amyegan** [reply]
If you can open a case at [vercel.com/help](https://www.vercel.com/help) with `Billing` as the problem area, it will be sent to the team that can with this 🙂

**Comment by manangouhari** [comment]
Yes! Awaiting help from the Vercel team. 

Hopefully they're able to sort it out, cause as I understand there's a chance that the domain might not be recoverable.
